tasty-golden-extra.cabal

```diff
@@ -31,17 +31,17 @@ source-repository head
 
 common deps
   build-depends:
-    , aeson           ==2.1.2.1    || ==2.2.3.0
+    , aeson           >=2.1.2.1    && <2.2.4
```

Comment:
These upper bounds are too tight, it is safe to allow newer minor versions because they aren't allowed to make breaking changes.